projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.spec.ts

```diff
@@ -28,7 +22,21 @@ describe('NavbarMobileMenuComponent', () => {
         fixture.detectChanges();
     });
 
-    it('should create', () => {
+    it('should create without IAppSwitcherService', () => {
```

Comment:
note that IAppSwitcherService is no longer needed. Although, if it is, the mobile menu will show the "all applications" button

Reply:
you're the man for writing tests for this